workspaces/lightspeed/packages/app/e2e-tests/utils/fileUpload.ts

```diff
@@ -36,6 +36,14 @@ export async function uploadFile(page: Page, filePath: string) {
   await fileChooser.setFiles(filePath);
 }
 
+export async function reUploadfile(page: Page, filePath: string) {
+  await uploadFile(page, filePath);
+  await expect(
+    page.getByRole('heading', { name: 'File upload failed' }),
+  ).toBeVisible();
+  await expect(page.getByText('File already exists.')).toBeVisible();
+}
```

Comment:
The name of this function doesn't describe it's behaviour. It sounds like a re-upload, but the function's logic is actually about verifying that uploading an existing file results in an error. So the pre-requisite for this function to work correctly is to upload the same file before calling this function, I would suggest choose a better name or even better you can warp them into meaningful helper function.

```ts
export async function uploadAndAssertDuplicate(page: Page, filePath: string, fileName: string) {
  await validateSuccessfulUpload(page, fileName);
  await uploadFile(page, filePath);
  await expect(
    page.getByRole('heading', { name: 'File upload failed' }),
  ).toBeVisible();
  await expect(page.getByText('File already exists.')).toBeVisible()
}
```